apps/volontulo/templates/offers/offer_apply.html

```diff
@@ -5,10 +5,11 @@
 {% block content-heading %}
 <div class="offer_view">
     <img class="img-responsive center-block" src="{{ MEDIA_URL }}{{ main_image|default:'' }}" alt="{{ offer.title|safe }}" />
+    {% if user.is_administrator %}
+        <a href="{% url 'offers_edit' offer.title|slugify offer.id %}" class="btn btn-primary">Edytuj ofertę</a>
+    {% endif %}
```

Comment:
I have removed offer edit button on offer apply earlier as it is the apply view not the offer itself. The edit button is for offer owner and admin, they shouldn't visit this page anyway with an intention to edit its offer. 

What do you think?